docs/DeveloperGuide.md

```diff
@@ -621,52 +621,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 3a. The given index is invalid.
-
-    * 3a1. InTrack shows an error message.
-
-      Use case resumes at step 2.
-
-**Use case: UC05 - Mark an internship application as completed**
-
-**MSS**
-
-1. User <u>views list of current internship applications (UC02)</u>.
-2. User requests to mark a specific internship application in the list as completed.
-3. InTrack marks the internship application as completed.
-
-   Use case ends.
-
-**Extensions**
-
-* 3a. The given index is invalid.
+* 2a The given index is invalid.
 
-    * 3a1. InTrack shows an error message.
+    * 2a1. InTrack shows an error message.
 
       Use case resumes at step 2.
-
+    
 *{More to be added}*
 
 ### Non-Functional Requirements
 
 1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2. Should be able to hold up to 1000 internship applications without a noticeable sluggishness in performance for typical usage.
 3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-4. Data should be stored consistently upon closing and reopening the app
-5. System should be usable by a Computing student who has never applied to an internship before
-6. System should be easily usable by the majority of Year 2+ Computing students
-
-
-*{More to be added}*
-
+4. Data should be stored consistently upon closing and reopening the app.
```

Comment:
Data is stored when it is changed though?

Reply:
what do you mean?

Reply:
Data is not stored when the app is closed or opened. As stated in the user guide, "data is saved in the hard disk automatically after any command that changes the data."

Reply:
I have updated the wording.